docs/resources/Channel.md

```diff
@@ -63,7 +30,123 @@ Represents a guild or DM channel within Discord.
+| parent_id                      | ?snowflake                                                                | id of the parent category for a channel (each parent category can contain up to 50 channels)                                                                                                    |
+| last_pin_timestamp?            | ?ISO8601 timestamp                                                        | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                                      |
+| default_auto_archive_duration? | integer                                                                   | default duration that the clients (not the API) will use for newly created threads, in minutes, to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080 |
+
+\* `rate_limit_per_user` also applies to thread creation. Users can send one message and create one thread during each `rate_limit_per_user` interval.
+
+\*\* News channels cannot have a `rate_limit_per_user` field - it is by virtue always 0 and isn't included.
+
+###### DM Channel
+
+| Field               | Type                                                      | Description                                                                                                                |
+|---------------------|-----------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| id                  | snowflake                                                 | the id of this channel                                                                                                     |
+| type                | integer                                                   | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types)                                                |
+| last_message_id     | ?snowflake                                                | the id of the last message sent in this channel (may not point to an existing or valid message)                            |
+| recipients          | array of [user](#DOCS_RESOURCES_USER/user-object) objects | the recipients of the DM                                                                                                   |
+| last_pin_timestamp? | ?ISO8601 timestamp                                        | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned. |
+
+###### Group Channel
```

Comment:
```suggestion
###### Group DM Channel
```
Also, worth considering whether this still needs to be documented.

Reply:
I agree - but I did not want to do any other changes than required to split up the channel object.

Reply:
GW group channels are still accessible to bots, so I don't see why it would be removed from the docs :shrug: